app/cli.py

```diff
@@ -155,6 +157,29 @@ def apikey():
         """apikey commands"""
         pass
 
+    @app.cli.command()
+    def check_bad_url():
+        """check for expired resource url"""
+        resources = Resource.query.all()
+        headers = {
+            'User-Agent': 'Mozilla/5.0 (Linux; Android 6.0; Nexus 5 Build/MRA58N) \
+                           AppleWebKit/537.36 (KHTML, like Gecko) \
+                           Chrome/79.0.3945.117 Mobile Safari/537.36'
+        }
+        processes = []
+
+        def print_bad_url(resource):
+            try:
+                res = requests.get(resource.url, headers=headers)
+                res.status_code > 400 and \
+                    print(f"resource_id: {resource.id} | resource_url: {resource.url}")
+            except Exception:
```

Comment:
Not the best idea to catch all exceptions. What if you lose internet connection? What if you write bad code? You'd only want to catch exceptions that indicate the resource is a goner.

Reply:
fixed.